draft-regext-rdap-x-media-type.md

```diff
@@ -189,11 +189,36 @@ Provisional Registration: No
 
 # Acknowledgements
 
-Pawel Kowalik provided ideas and feedbacks that have contributed to
+Pawel Kowalik and James Mitchel have provided ideas and feedbacks that have contributed to
 the content of this document.
 
 {backmatter}
 
+# Using the Vary Header {#vary_header}
+
+Server implementers may want to consider using the `vary` header depending on the caching
```

Comment:
may -> MAY?

Reply:
I don't think this is normative and therefore the MAY should be lowercase.